alf/experience_replayers/replay_buffer.py

```diff
@@ -118,6 +119,9 @@ def __init__(self,
                 Usually consistent with ``TDLoss.gamma``.
             reward_clip (tuple|None): None or (min, max) for reward clipping.
             enable_checkpoint (bool): whether checkpointing this replay buffer.
+            convert_only_minibatch_to_device (bool): when True, only convert a minibatch
```

Comment:
It feels weird that this kind of setting is at particular DataTransformer. It should be at the level of Algorithm. And if GPU memory is a concern, now we use fractional unroll_length to reduce it. So this option is not that necessary.